app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.setdefault(name, self)
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for people_dict in people:
+        person = Person(people_dict["name"], people_dict["age"])
+        person_list.append(person)
+
+    for ind, spouses in enumerate(people):
+        if "wife" in spouses and spouses["wife"] is not None:
+            wife = Person.people.get(spouses["wife"])
+            person_list[ind].wife = wife
+            wife.husband = person_list[ind]
+        if "husband" in spouses and spouses["husband"] is not None:
```

Comment:
is not None should be omitted, it will work without it

Reply:
But it doesn't work)